Code/GraphMol/FMCS/FMCS.h

```diff
@@ -44,6 +44,7 @@ struct RDKIT_FMCS_EXPORT MCSAtomCompareParameters {
   bool RingMatchesRingOnly = false;
   bool CompleteRingsOnly = false;
   bool MatchIsotope = false;
+  float MaxDistance = -1.0;
```

Comment:
Nit: we generally use doubles unless there's a particular reason not to

Reply:
fixed